access/ms-teams/validate.go

```diff
@@ -134,3 +117,28 @@ func validate(configPath, userID string) error {
 
 	return nil
 }
+
+func loadConfigAndCheckApp(configPath string) (*Bot, *Config, error) {
+	c, err := LoadConfig(configPath)
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+
+	fmt.Printf(" - Checking application %v status...\n", c.MSAPI.TeamsAppID)
+
+	b, err := NewBot(c.MSAPI, "local", "")
+	if err != nil {
+		return nil, nil, trace.Wrap(err)
+	}
+
+	teamApp, err := b.GetTeamsApp(context.Background())
```

Comment:
I would pass context from the caller to `loadConfigAndCheckApp` method and use it here.

Reply:
Done